internal/server/v0/internalplace/golden/get_place_page_data/asm.sample.json

```diff
@@ -699,334 +699,88 @@
-            "1995-08": 5,
-            "1995-09": 2,
-            "1995-10": 1,
-            "1996-07": 3,
-            "1996-08": 3,
-            "1996-09": 2,
-            "1997-08": 5,
-            "1998-07": 1,
-            "1998-08": 3,
-            "1998-09": 2,
-            "1999-04": 1,
-            "1999-06": 1,
-            "1999-07": 1,
-            "1999-08": 3,
-            "1999-09": 3,
-            "1999-10": 1,
-            "2000-07": 2,
-            "2000-08": 3,
-            "2000-09": 1,
-            "2001-06": 1,
-            "2001-07": 4,
-            "2001-08": 2,
-            "2001-09": 1,
-            "2002-07": 1,
-            "2002-08": 2,
-            "2002-09": 3,
-            "2003-07": 2,
-            "2003-08": 3,
-            "2003-09": 1,
-            "2004-06": 1,
-            "2004-07": 2,
-            "2004-08": 2,
-            "2004-09": 1,
-            "2005-07": 2,
-            "2005-08": 3,
-            "2005-09": 3,
-            "2006-05": 1,
-            "2006-06": 1,
-            "2006-07": 3,
-            "2006-08": 3,
-            "2006-09": 1,
-            "2007-07": 1,
-            "2007-08": 2,
-            "2007-09": 2,
-            "2007-10": 1,
-            "2008-04": 1,
-            "2008-06": 1,
-            "2008-07": 2,
-            "2008-08": 2,
-            "2008-09": 2,
-            "2009-06": 2,
-            "2009-07": 3,
-            "2009-08": 2,
-            "2009-09": 2,
-            "2009-10": 2,
-            "2010-07": 1,
-            "2010-08": 1,
-            "2010-09": 2,
-            "2010-10": 3,
-            "2011-06": 2,
-            "2011-07": 2,
-            "2011-08": 1,
-            "2011-09": 2,
-            "2012-06": 1,
-            "2012-07": 1,
-            "2012-08": 5,
-            "2012-09": 1,
-            "2013-06": 1,
-            "2013-07": 3,
-            "2013-08": 4,
-            "2013-09": 1,
-            "2013-10": 1,
-            "2013-11": 1,
-            "2014-06": 1,
-            "2014-07": 2,
-            "2014-08": 1,
-            "2014-09": 3,
-            "2015-06": 1,
-            "2015-07": 1,
-            "2015-08": 2,
-            "2015-09": 1,
-            "2015-10": 1,
-            "2016-05": 1,
-            "2016-07": 2,
-            "2016-08": 2,
-            "2016-09": 2,
-            "2016-10": 2,
-            "2017-06": 1,
-            "2017-07": 3,
-            "2017-08": 2,
-            "2017-09": 3,
-            "2017-10": 1,
-            "2018-06": 1,
-            "2018-07": 4,
-            "2018-08": 4,
-            "2018-09": 2,
-            "2019-07": 1,
-            "2019-08": 3,
-            "2019-09": 2,
-            "2020-06": 1,
-            "2020-08": 5,
-            "2020-09": 2,
-            "2020-10": 1,
-            "2021-06": 1,
-            "2021-07": 3,
-            "2021-08": 1,
-            "2021-10": 2,
-            "2022-07": 1,
-            "2022-08": 2,
-            "2022-09": 1
+            "1945": 5,
```

Comment:
Is this expected?

Reply:
yes, with the last release it went from P1Y -> P1M 
/cc @juliawu